functions-and-operators/tidb-functions.md

```diff
@@ -102,6 +102,29 @@ select tidb_decode_key('7480000000000000FF3E5F720400000000FF0000000601633430FF33
 1 row in set (0.001 sec)
 ```
 
+The first region of a table will start with a key that only has the `table_id` of the table. The last region of the table ends with a `table_id` of the next table. Any regions in between will have longer keys that includes a `_tidb_rowid` or `handle`.
+
+{{< copyable "sql" >}}
+
+```sql
+SELECT TABLE_NAME, START_KEY, END_KEY FROM information_schema.TIKV_REGION_STATUS WHERE TABLE_NAME='stock' AND IS_INDEX=0;
+```
+
+```sql
++------------+--------------------------------------------------------+--------------------------------------------------------+
+| TABLE_NAME | START_KEY                                              | END_KEY                                                |
++------------+--------------------------------------------------------+--------------------------------------------------------+
+| stock      | 7480000000000000FF4700000000000000F8                   | 7480000000000000FF475F728000000000FF029DF00000000000FA |
+| stock      | 7480000000000000FF475F728000000000FF0D7ECA0000000000FA | 7480000000000000FF4900000000000000F8                   |
```

Comment:
Should the END_KEY be 7480000000000000FF4800000000000000F8 (i.e. table_id + 1)? Internally I would assume we are skipping an id, effectively doing table_id + 2?

Reply:
```
sql> SELECT DB_NAME, TABLE_ID MOD 2, COUNT(*) FROM information_schema.TIKV_REGION_STATUS WHERE TABLE_ID GROUP BY DB_NAME, TABLE_ID MOD 2 ORDER BY 1,2;
+--------------------+----------------+----------+
| DB_NAME            | TABLE_ID MOD 2 | COUNT(*) |
+--------------------+----------------+----------+
| INFORMATION_SCHEMA |              0 |       44 |
| INFORMATION_SCHEMA |              1 |       45 |
| METRICS_SCHEMA     |              0 |      320 |
| METRICS_SCHEMA     |              1 |      320 |
| PERFORMANCE_SCHEMA |              0 |       16 |
| PERFORMANCE_SCHEMA |              1 |       18 |
| mysql              |              0 |      112 |
| mysql              |              1 |        6 |
| test               |              0 |       19 |
+--------------------+----------------+----------+
9 rows in set (0.0071 sec)
```
Here the `test` schema only has even table_id's.

Reply:
Reported as https://github.com/pingcap/tidb/issues/48725.